modules/nw-ptp-operator-metrics-reference.adoc

```diff
@@ -38,18 +62,18 @@ Possible values for PTP clock state are `FREERUN`, `LOCKED`, or `HOLDOVER`.
 |`openshift_ptp_frequency_adjustment_ns`
```

Comment:
openshift_ptp_clock_state, openshift_ptp_delay_ns, openshift_ptp_frequency_adjustment_ns,openshift_ptp_interface_role,openshift_ptp_max_offset_ns,
openshift_ptp_offset_ns,openshift_ptp_process_status
 available for all config not just for T-GM

Reply:
These are the metrics that are `additionally` available only when T-GM is configured 
1. openshift_ptp_nmea_status
2. openshift_ptp_phase_status
3. openshift_ptp_pps_status
4. openshift_ptp_gnss_status

Reply:
Updated as per the suggestion.

Reply:
Updated as per the suggestion.